libs/ui/src/components/TheImage/TheImage.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <component :is="imageComponent" v-bind="attrs" />
+</template>
+
+<script lang="ts" setup>
+import type {
+  ComputedOptions,
+  ConcreteComponent,
+  ImgHTMLAttributes,
+  MethodOptions,
+  ReservedProps,
+} from 'vue'
+import omit from 'lodash/omit'
```

Comment:
There is no lodash in `libs/ui`

Reply:
Is there any way to configure the environment to see errors when using packages in libs/ui that are not installed there?